hat/assets/js/apps/Iaso/domains/instances/components/InstancesFiltersComponent.js

```diff
@@ -358,6 +362,15 @@ const InstancesFiltersComponent = ({
                         label={MESSAGES.org_unit_type_id}
                         loading={fetchingOrgUnitTypes}
                     />
+                    <InputComponent
+                        type="select"
+                        multi
+                        keyValue="planningIds"
+                        onChange={handleFormChange}
+                        value={formState.planningIds.value || null}
+                        options={planningsDropdownOptions}
+                        label={MESSAGES.planning}
```

Comment:
please add loading state while loading `planningsDropdownOptions`. It will display a load in the dropdown.
Also the value can be empty while loading.
See other implementations for example